taskcluster/ci/ui-test/kind.yml

```diff
@@ -0,0 +1,61 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: focus_android_taskgraph.loader.multi_dep:loader
+
+transforms:
+    - focus_android_taskgraph.transforms.multi_dep:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
```

Comment:
@sarah-clements The APKs are not being found by Flank. The job-defaults declared in the kind file map the APK file names. The kind file in the Fenix project provides an example of how they are used and setup.  

https://github.com/mozilla-mobile/fenix/blob/6a3adb5aeba976a7146eb96a2ef3794fae909304/taskcluster/ci/ui-test/kind.yml#L21-L31